internal/pkg/api/repository/result/mongo.go

```diff
@@ -59,34 +58,57 @@ func (r *MongoRepository) GetExecutions(ctx context.Context, filter Filter) (res
 func (r *MongoRepository) GetExecutionTotals(ctx context.Context, filter Filter) (result testkube.ExecutionsTotals, err error) {
 
 	query, _ := composeQueryAndOpts(filter)
-	query["scriptType"] = bson.M{"$exists": true}
 	total, err := r.Coll.CountDocuments(ctx, query)
 	if err != nil {
-		return
+		return result, err
 	}
 	result.Results = int32(total)
 
-	query["executionResult"] = bson.M{"status": testkube.QUEUED_ExecutionStatus}
-	queued, err := r.Coll.CountDocuments(ctx, query)
-	if err != nil {
-		return
-	}
-	result.Queued = int32(queued)
-
-	query["executionResult"] = bson.M{"status": testkube.PENDING_ExecutionStatus}
-	pending, err := r.Coll.CountDocuments(ctx, query)
-	if err != nil {
-		return
+	if status, ok := query["executionresult.status"]; ok {
```

Comment:
@nicufk could be done more efficient by MongoDB aggregate in one mongo query - without many hits to database. please add tech-debt task for later for this

Reply:
Done #522